examples/dave_demo_launch/launch/dave_oberon7_moveit.launch

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0"?>
 <launch>
-	<!-- <arg name="world_name" default="worlds/empty.world"/> -->
-	<arg name="world_name" default="$(find dave_worlds)/worlds/dave_ocean_waves.world"/>
+	<arg name="world_name" default="$(find dave_worlds)/worlds/dave_bimanual_example.world"/>
 	<arg name="joy_id" default="0"/>
 	<arg name="paused" default="false"/>
 	<arg name="gui" default="true"/>
+	<arg name="effort_controller" default="true"/>
```

Comment:
By having two separate boolean variables to select controllers, this launch file will try to end up spawning the same controller manager twice which will lead to an exception being thrown if the users launches the file with `position_controller:=true` and not explicitly passing `effort_controller:=false`. See below.

```
yadu@yadu:~/ws_dave$ roslaunch dave_demo_launch dave_oberon7_moveit.launch position_controller:=true
... logging to /home/yadu/.ros/log/2745cdaa-aa96-11ec-883e-e9d7e55577dc/roslaunch-yadu-112151.log
Checking log directory for disk usage. This may take a while.
Press Ctrl-C to interrupt
Done checking log file disk usage. Usage is <1GB.

RLException: Invalid <arg> tag: arg 'controller_type' has already been declared. 

Arg xml is <arg name="controller_type" value="position" if="$(arg position_controller)"/>
The traceback for the exception was written to the log file
```

One way to avoid this problem (and I think you're already doing this in `upload_rexrov_oberon7_moveit.launch`), is to have a single `controller_type` argument here with default value `effort`. We can directly pass this argument when we include `upload_rexrov_oberon7_moveit.launch`. The `goup if` block below where we spawn the controller_manager can also just check for this string value.

Reply:
@Yadunund Thank you for the catch. I've modified the file to accept a single `controller_type` arg as per your suggestion.